### PR TITLE
remove both gsRadarX and gsRadarY, their both always RADAR_WINDOW_X a…

### PR DESCRIPTION
**Pull request by derSteFfi**
…nd RADAR_WINDOW_TM_Y repsectively

### CODE REVIEW

File: Build/TileEngine/Radar_Screen.cc [hunk 1]
```diff
@@ -278,10 +269,10 @@ void RenderRadarScreen()
 			INT16 const sHeight = RADAR_WINDOW_HEIGHT;
 			INT16 const sX      = RADAR_WINDOW_X;
 
-			INT16 const sRadarTLX = sTopLeftWorldX     * gdScaleX - sRadarCX + sX       + sWidth  / 2;
-			INT16 const sRadarTLY = sTopLeftWorldY     * gdScaleY - sRadarCY + gsRadarY + sHeight / 2;
-			INT16 const sRadarBRX = sBottomRightWorldX * gdScaleX - sRadarCX + sX       + sWidth  / 2;
-			INT16 const sRadarBRY = sBottomRightWorldY * gdScaleY - sRadarCY + gsRadarY + sHeight / 2;
+			INT16 const sRadarTLX = sTopLeftWorldX     * gdScaleX - sRadarCX + sX      					 + sWidth  / 2;
```
Comment (selaux) [inline Build/TileEngine/Radar_Screen.cc]: Is this intended?

File: Build/TileEngine/Radar_Screen.cc [hunk 2]
```diff
@@ -475,7 +466,7 @@ static void CreateDestroyMouseRegionsForSquadList(void)
 static void RenderSquadList(void)
 {
 	INT16 const dx = RADAR_WINDOW_X;
-	INT16 const dy = gsRadarY;
```
Comment (selaux) [inline Build/TileEngine/Radar_Screen.cc]: Couldn't it be something else than `RADAR_WINDOW_TM_Y` here?

File: Build/TileEngine/Radar_Screen.cc (hunk 1 above)
Comment (derSteFfi) [inline Build/TileEngine/Radar_Screen.cc]: yes. i Had to adjust the tabbing to reflect hte new length of the lines. Looks fine when viewed with 2 spaces as tabs.

File: Build/TileEngine/Radar_Screen.cc (hunk 2 above)
Comment (derSteFfi) [inline Build/TileEngine/Radar_Screen.cc]: theoretically, yes. But gsRadarY never gets assigned with anything else...

**Pull request merged**